src/components/chip/chip.scss

```diff
@@ -0,0 +1,16 @@
+.chip-container {
+  display: inline-flex;
+  flex-direction: row;
+  background-color: #7000FF;
```

Comment:
Suggestion: We probably will want to set this styling inline to make dynamic.

We need to get different colors based on the category.

Reply:
Sounds good! Happy to re-work this and make the styling in-line and dynamic :)